src/ui/public/styles/spinner.less

```diff
@@ -1,39 +1,6 @@
-@import "~ui/styles/variables";
-
-@size: 400px;
-@color1: @kibanaPink1;
-@color2: @kibanaPink2;
-
 .spinner.ng-hide {
```

Comment:
I looked for `class="spinner"` and found several instances. If you just search for `spinner` there is another place it's used in conjunction with another class, and several cases where it's being styled in a LESS file.